src/accelerate/commands/config/config_args.py

```diff
@@ -156,10 +156,13 @@ def __post_init__(self):
 class SageMakerConfig(BaseConfig):
     ec2_instance_type: str
     iam_role_name: str
+    image_uri: str
```

Comment:
I think we should default to the HF Image?

Reply:
HF image is used by default if `image_uri` isn't specified based on the `Python`, `PyTorch` and `Transformers` versions.